deploy/clowdapp.yaml

```diff
@@ -788,3 +788,5 @@ parameters:
   value: "30"
 - name: MAX_LOADED_ADVISOR_LISTENER_MSGS
   value: "30"
+- name: SYNC_ON_START
```

Comment:
you need to set this value for vmaas-sync above, otherwise it's not used

Reply:
forgot, fixed